README.md

```diff
@@ -502,6 +503,22 @@ custom:
 This can be useful, in case you want to upload the source maps to your Error
 reporting system, or just have it available for some post processing.
 
+#### Try native zip functionality
+
+Native zip is much faster than node zip, however the "bestzip" library used lacks
```

Comment:
From my testing it also appears that sometimes it will be very helpful to have the `excludeRegex` setting work. Having files like `.DS_Store` or `.yarn-integrity` removed will make the ZIP files even more stable. Maybe this is the additional argument for using `archiver` in place of `bestzip`, as the former one has the `glob()` method?